nx-X11/programs/Xserver/hw/nxagent/Rootless.c

```diff
@@ -351,14 +351,14 @@ void nxagentRootlessRestack(unsigned long children[], unsigned int nchildren)
 
   for (i = 0; i < ntoplevel; i++)
   {
-    fprintf(stderr, "[%p]\n", toplevel[i]);
+    fprintf(stderr, "nxagentRootlessRestack: [%p]\n", toplevel[i]);
```

Comment:
I'd use `"%s: ...", __func__, ...` in this commit and especially in all new code.